app/models/instructeur.rb

```diff
@@ -134,12 +134,7 @@ def notifications_for_dossier(dossier)
   end
 
   def notifications_for_groupe_instructeurs(groupe_instructeurs)
-    Dossier
-      .visible_by_administration
-      .not_archived
-      .where(groupe_instructeur: groupe_instructeurs)
-      .merge(followed_dossiers)
-      .with_notifications
+    notifications_for_condition(groupe_instructeur: groupe_instructeurs)
```

Comment:
Est ce qu'on peut dégager le `for_condition` , dans ma tête ça me donne l'impression que c'est en lien avec le conditionnel

```suggestion
    notifications_for(groupe_instructeur: groupe_instructeurs)
```

Reply:
yes, amendé